village/src/App.js

```diff
@@ -11,14 +12,45 @@ class App extends Component {
       smurfs: [],
     };
   }
+
+  componentDidMount() {
+    axios
+      .get('http://localhost:3333/smurfs')
+      .then(response => {
+        console.log ("this is the response:", response);
+        this.setState({smurfs: response.data }, () => console.log(this.state.smurfs));
+      })
+      .catch(error => {
+        console.error('Server Error', error);
+      });
+  }
   // add any needed code to ensure that the smurfs collection exists on state and it has data coming from the server
   // Notice what your map function is looping over and returning inside of Smurfs.
   // You'll need to make sure you have the right properties on state and pass them down to props.
   render() {
     return (
       <div className="App">
-        <SmurfForm />
-        <Smurfs smurfs={this.state.smurfs} />
+        {/* <SmurfForm />
+        <Smurfs smurfs={this.state.smurfs} /> */}
+
+        <header>
+          <h1>Welcome to the Smurf Village!</h1>
+          <h3>Join us in the Smurf Village here:</h3>
+          <Link to='/smurfs'>Smurf Village</Link>
+          <h3>Add a new member to our Smurf Village here:</h3>
+          <Link to='/smurfform'>New Smurf</Link>
+        </header>
+
+        <br></br>
+        <br></br>
+        <br></br>
+        <br></br>
+        <br></br>
+        <br></br>
+        
+        <Route path='/smurfform' component={SmurfForm} /> 
+        <Route path='/smurfs' render={props => <Smurfs {...props} smurfs={this.state.smurfs} />} />
```

Comment:
You don't need to spread the props `{...props}` to the `Smurfs` component in this context. Spreading the props gives you access to history, location, and match, which you aren't using in `Smurfs` or it's child component `Smurf`. The only prop you need that you're using is `smurfs={this.state.smurfs}`.